matter/wifi/rs911x/rsi_if.c

```diff
@@ -318,27 +326,23 @@ static void wfx_rsi_do_join(void)
      * Right now it's done by hand - we need something better
      */
     wfx_rsi.dev_state |= WFX_RSI_ST_STA_CONNECTING;
-    while (((status = rsi_wlan_connect_async((int8_t *)&wfx_rsi.sec.ssid[0],
-                                             (rsi_security_mode_t)wfx_rsi.sec.security,
-                                             &wfx_rsi.sec.passkey[0],
-                                             wfx_rsi_join_cb))
-            != RSI_SUCCESS)
-           && (++wfx_rsi.join_retries < WFX_RSI_CONFIG_MAX_JOIN)) {
+    while (((status = rsi_wlan_connect_async(
+                 (int8_t *)&wfx_rsi.sec.ssid[0],
+                 (rsi_security_mode_t)wfx_rsi.sec.security,
+                 &wfx_rsi.sec.passkey[0], wfx_rsi_join_cb)) != RSI_SUCCESS) &&
+           (++wfx_rsi.join_retries < WFX_RSI_CONFIG_MAX_JOIN)) {
       wfx_rsi.dev_state &= ~WFX_RSI_ST_STA_CONNECTING;
-      WFX_RSI_LOG("%s: rsi_wlan_connect_async failed with status: %02x on try %d",
-                  __func__,
-                  status,
-                  wfx_rsi.join_retries);
+      WFX_RSI_LOG(
+          "%s: rsi_wlan_connect_async failed with status: %02x on try %d",
+          __func__, status, wfx_rsi.join_retries);
       vTaskDelay(4000);
       /* TODO - Start a timer.. to retry */
     }
-    if (wfx_rsi.join_retries == 5) {
+    if (wfx_rsi.join_retries == JOIN_RETRIES_CNT_5) {
```

Comment:
This is ok was the Name tells me why the values means. You don't always have to Specify in the define name what the value is.  `MAX_JOIN_RETRIES_COUNT` would or something I like would be great